relevanceai/dataset_api/dataset.py

```diff
@@ -58,7 +59,7 @@ def encode_documents(documents):
         self.client.pull_update_push(self.dataset_id, encode_documents)
 
 
-class Dataset:
+class Dataset(BatchAPIClient):
```

Comment:
Why does Dataset inherit from BatchAPIClient, when it has an internal variable self.client?

Reply:
Removed :) thanks